app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/Tooltip.kt

```diff
@@ -102,52 +105,53 @@ private fun InnerChatTooltip(
   modifier: Modifier = Modifier,
   maxWidth: Dp = TooltipDefaults.defaultMaxWidth,
 ) {
-  val generalOffsetY = when (beakDirection) {
-    BottomCenter, BottomStart, BottomEnd -> arrowHeightDp
-    TopCenter, TopStart, TopEnd -> -arrowHeightDp
-    Start -> 0.dp
-    End -> 0.dp
-  }
-  val generalOffsetX = 0.dp
-  Box(
-    modifier
-      .widthIn(max = maxWidth)
-      .wrapContentHeight(Alignment.Top, true)
-      .wrapContentWidth(Alignment.End, true)
-      .offset(y = generalOffsetY, x = generalOffsetX),
-  ) {
-    Crossfade(show, label = "tooltip") { crossfadeShow ->
-      if (crossfadeShow) {
-        val shape = TooltipDefaults.shape
-        Surface(
-          onClick = onClick,
-          color = tooltipStyle.containerColor,
-          shape = remember(shape) { shape.withBeak(beakDirection) },
+  val shape = TooltipDefaults.shape
+  val density = LocalDensity.current
+  // Save the height before the minimumInteractiveComponentSize is applied, so that the layout can take up only the
+  // space it needs, but still let the touch size be accessible
+  var knownHeight by remember { mutableStateOf(0.dp) }
+  Crossfade(
+    targetState = show,
+    label = "tooltip",
+    modifier = modifier,
+  ) { crossfadeShow ->
+    if (crossfadeShow) {
+      Surface(
+        color = tooltipStyle.containerColor,
+        shape = remember(shape) { shape.withBeak(beakDirection) },
+        modifier = Modifier
+          .widthIn(
+            min = TooltipDefaults.defaultMinWidth,
+            max = maxWidth,
+          )
+          .height(knownHeight)
+          .wrapContentHeight(unbounded = true)
+          .minimumInteractiveComponentSize()
+          .clickable(onClick = onClick)
+          .onPlaced { knownHeight = with(density) { it.size.height.toDp() } },
```

Comment:
some magic here:))

Reply:
Some magic, which may absolutely not be needed after all.
![image](https://github.com/user-attachments/assets/f1eaee0b-813d-41ff-85c3-14ec359ce49c)

Here's what I was trying to do, and the answer I received. I will try without it too. It seems I may have had a wrong understanding of what makes the clickable modifier make the touch size extend the bounds of the item itself. I thought minimumInteractiveComponentSize was needed, but perhaps not. Which would make all this weird height stuff go away!

Reply:
Okay! So it has the right interactive size by default, and minimumInteractiveComponentSize is only needed if we have more than 1 interactive component close to each other?

Reply:
Yup, totally unneeded, I was just wrong to try and use `minimumInteractiveComponentSize`.

I replied there too
![image](https://github.com/user-attachments/assets/1d481d30-bfd7-4ade-8e2c-b6a6bb4a2d0d)